src/pages/ViewInterviewPage.tsx

```diff
@@ -94,6 +94,9 @@ const InterviewView = ({ interview }: { interview: HydratedInterview }) => {
         <ModalContent maxH="800px" maxW="700px">
           <ModalCloseButton />
           <ReportPopUp
+            title="Report Interview"
+            badgeColorScheme="red"
+            reasonPlaceholder="Enter reason for reporting this interview"
```

Comment:
Your reason for all of them is "Enter reason for reporting this <report type>". We can probably get rid of this and just fill in the report type part with your report type.

Reply:
can make it default prop